packages/mobile/src/transactions/contract-utils.ts

```diff
@@ -0,0 +1,228 @@
+import { values } from 'lodash'
```

Comment:
Can this be renamed to `send-utils`? I realize the name in walletKit was contract-utils but that's not really accurate

Reply:
My feeling is that our send plumbing is still pretty ugly and could use some refactoring, which would hopefully remove a few of these files and function indirection. What do you think?

Reply:
Agreed, could be better. Ideally we'd just rely on contract kit and extend it's capabilities to support any logging we need